indigo_app/static/javascript/indigo/views/document_editor.js

```diff
@@ -576,6 +578,11 @@
           .fail(fail);
       }
 
+      // TODO: a better way of reloading the page (will redirect to provision chooser for now)
+      if (Indigo.Preloads.provisionEid !== Indigo.Preloads.newProvisionEid) {
```

Comment:
this will change to check the expected against the actual in the XML, rather than using preloads.newProvisionEid